modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -136,6 +136,13 @@ API assets, published by Exchange, are the components that comprise applications
 
 For details about sharing assets via a private Exchange or an Exchange public portal, see xref:exchange::about-sharing-assets.adoc[Share Assets].
 
+=== API Console
+
+When you create or edit APIs in API Manager, use xref:apikit::apikit-simulate[API Console] to view and test your API specification. You can test RAML and OAS APIs when you manage the API as an endpoint with proxy. The console flow is already included and enabled (by default) in the API specification when you download an out-of-the-box proxy for your Mule application.
+
+Starting with Mule 4, you can enable access to API Console and modify the API Console URL from API Manager. API Console access from API Manager is disabled by default. For information about how to enable API Console access from API Manager, see xref:manage-exchange-api-task.adoc[Creating an API Instance by Importing from Exchange].
```

Comment:
Instead of _API Console URL_, I would say _API Console Path_

Reply:
Done! Thanks!